src/js/views/wrappers/clickable.js

```diff
@@ -9,14 +9,19 @@ module.exports = Component => {
             };
         }
 
-        toggleClicked() {
+        toggleClicked(clickedNode) {
+            // Don't toggle the row if a link was clicked.
+            if (clickedNode.target.nodeName === 'A') {
```

Comment:
In addition to A, there are a couple places on the roster with a similar problem that could be checked here: the PT column (Select) and the Release column (Button). Could that also be solved the same way?